### PR TITLE
refactor: move submissions to base serializer class

### PR DESCRIPTION
**Pull request by shawnsarwar**
To improve performance move Submissions to a serialzer based not based on ModelSerializer.

### CODE REVIEW

File: aether-kernel/aether/kernel/api/serializers.py [hunk 1]
```diff
@@ -226,12 +230,12 @@ def create(self, validated_data):
             raise serializers.ValidationError(
                 {'mappingset': [_('Mapping set must be provided on initial submission')]}
             )
-
-        return super(SubmissionSerializer, self).create(validated_data)
+        try:
+            return models.Submission.objects.create(**validated_data)
+        except Exception as e:                      # pragma: no cover : don't know how to trigger
+            raise serializers.ValidationError(e)    # without first triggering VE on the serializer
```
Comment (obdulia-losantos) [inline aether-kernel/aether/kernel/api/serializers.py:236]: :joy: 

Add an unknown property :thinking: 

```python
        submission_bad = serializers.SubmissionSerializer(
            data={
                'mappingset': mappingset.data['id'],
                'project': project.data['id'],
                'payload': EXAMPLE_SOURCE_DATA,
                'is_extracted': True,
                'who_am_i': 'oops',
            },
            context={'request': self.request},
        )
```

Comment (shawnsarwar) [reply aether-kernel/aether/kernel/api/serializers.py:236]: @obdulia-losantos  No luck. That doesn't raise an exception, which is concerning...

Comment (obdulia-losantos) [reply aether-kernel/aether/kernel/api/serializers.py:236]: :flushed:

**Review by obdulia-losantos — APPROVED** [review]

File: aether-kernel/aether/kernel/api/serializers.py (hunk 1 above)
Comment (shawnsarwar) [reply aether-kernel/aether/kernel/api/serializers.py:236]: After reflection, we are choosing to ignore extra fields and not spend compute time validating things that are ignored by the serializer and Model.

**Pull request merged**